.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
why it is needed?

Reply:
My VSCode was using it's own typescript version instead of the workspace version. This makes sure VSCode will use the one on `node_modules` instead

Reply:
But we shouldn't add personal editor config to the code, other peoples can use another IDE

Reply:
This is odd to me. I always see this on other people's code and this makes total sense to me. The idea is that, for everyone with VSCode, the right version of typescript will be used, not the global, default one. But I'll remove since other people find this weird.